src/commands/Miscellaneous/discordstatus.ts

```diff
@@ -15,27 +15,121 @@ export default class extends SteveCommand {
 
 	public async run(msg: KlasaMessage): Promise<Message> {
 		const url = 'https://srhpyqt94yxb.statuspage.io/api/v2/summary.json';
-		const embedData = msg.language.tget('commandDiscordStautsEmbed');
-
-		return fetch(url, { method: 'Get' })
-			.then(res => res.json())
-			.then(json => {
-				const embed = new MessageEmbed()
-					.setTitle(json.status.description)
-					.setDescription(embedData.decription(json.status.indicator))
-					.addFields(json.components.map((component: { name: any; status: any }) => ({
+		const embedData = msg.language.tget('commandDiscordStatusEmbed');
+
+		const currentStatus = await fetch(url, { method: 'Get' }).then(r => r.json()) as DiscordStatus;
+
+		if (!currentStatus) {
+			return msg.channel.send(msg.language.tget('commandDiscordStatusError'));
+		}
+
+		const fields: Array<EmbedField> = [];
```

Comment:
`EmbedField[]` instead of `Array<EmbedField>`

Reply:
Resolved by [029eebb](https://github.com/tuataria/steve/pull/63/commits/029eebb0e89e4770e99b06fc6556b9367f350771)